tests/fuzz/Dockerfile.fuzz

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Istio Authors
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specif
+
+FROM golang:1.15
```

Comment:
This may cause issues in the future, the whole repo is not compile-able with 1.15. But I did see those issues with 1.16 that I haven't yet resolved.